cmd/panfs-multipart.go

```diff
@@ -74,6 +74,14 @@ func (fs *PANFSObjects) decodePartFile(name string) (partNumber int, etag string
 
 // Appends parts to an appendFile sequentially.
 func (fs *PANFSObjects) backgroundAppend(ctx context.Context, bucket, object, uploadID string) {
+	fs.multipartTmpFolderMutex.RLock()
+	tmpDir, ok := fs.multipartTmpFolder[uploadID]
```

Comment:
uploadID is a random UUID. Just an idea, maybe we can have static rule for calculating path to background append as:

```
pathJoin(bucketDir, panfsS3TmpDir, fs.fsUUID, hash(uploadID) % tmpDirCount)
```

random UUID should get nice distribution across temporary folders and we will not need to track everything in map and have a mutex. In addition, we always should be able to find temporary file for upload id even without minio runtime context.

Reply:
Removed multipart changes